src/prefs/prefs-gtk3.ts.old

```diff
@@ -48,9 +48,7 @@ export function buildGtk3() {
     theme.addSetting('panel-icon-color', WidgetType.BOOLEAN);
     theme.addSetting('taskbar-item-style', WidgetType.COMBO);
     theme.addSetting('surface-opacity', WidgetType.INT);
-    if (!ShellVersionMatch('3.34')) {
-        theme.addSetting('blur-background', WidgetType.BOOLEAN);
```

Comment:
This shouldn't have been removed? Right?

Reply:
indeed

Reply:
But this file is garbage anyway